README.md

```diff
@@ -1,2 +1,87 @@
-# dev-team
-Proposal for the Dev Team
+# Student Robotics Dev Team Proposal
+
+This document aims to solve the important issue of development by outlining the formation of a number of smaller sub-groups within Student Robotics:
+- The Development Coordination Team.
+- A number of _project groups_.
+
+## The Development Coordination team
+
+The Development Coordination team (a.k.a. the _Dev Team_) is a group of volunteers that are chosen annually in a similar fashion to the formation of the _Core Team_, although they may be responsible for projects beyond the annual timescale.
+
+The goals of the _Dev Team_ are to ensure the long-term longevity of Student Robotics by organising projects that enhance the experience for SR’s competitors and volunteers in line with its values, and enable SR’s long term growth. Additionally they will aim to fulfill the values of Student Robotics, in particular our goal of reflecting reality by incorporating the latest technology whenever feasible.
```

Comment:
> reflecting reality by incorporating the latest technology whenever feasible.

I disagree that "reflecting reality" leads to "incorporating the latest technology". While incorporating newer technologies has benefits, I would propose that there are much stronger engineering principles which are more reflective of reality and may run contrary to this. The main example I would cite is using the right tool for any given job, which may include fitting in well with an existing environment.

This does _not_ mean that we should constrain the Dev Team to legacy systems, however it does mean that they should consider how any new system will interact with those systems.

Reply:
Separately, there is also an argument that existing systems are the environment that most engineers operate in, so always presenting competitors with the latest bleeding-edge stuff may actually move away from reality. Clearly there is a balance to be struck between following the wider trends of industry and putting us in a situation where the Dev Team is incentivised to rebuild our stack ever few years by chasing the latest fad.

Reply:
I agree that using the latest technology rarely reflects reality. In my experience (both professionally and not), large projects will tend towards the LTS model of technologies. Intentionally _not_ using the latest versions of technology, and instead using something which is slightly older, with the added benefit of stability and support. 

Given a recent example, the latest _feasible_ version of the Linux kernel is 5.0, but I suspect its market share is drastically less than 0.1%.  

The entire final sentence actually duplicates part of the former, which already talks about the values of SR. It seems it can be removed without losing any information about the role of the dev team to SR.

```suggestion
The goals of the _Dev Team_ are to ensure the long-term longevity of Student Robotics by organising projects that enhance the experience for SR’s competitors and volunteers in line with its values, and enable SR’s long term growth.
```

Reply:
```suggestion
The goals of the _Dev Team_ are to ensure the long-term longevity of Student Robotics by organising projects that enhance the experience for SR’s competitors and volunteers in line with its values, and enable SR’s long term growth. Additionally they will aim to fulfill the values of Student Robotics, in particular our goal of reflecting reality by incorporating the latest stable technology whenever reasonable. 
```

Reply:
I don't think we should have the word "latest", maybe "industry standard"?

If anything, we're doing better at reflecting reality by having horrible out-dated systems which are too expensive to replace, as that's what really happens in software engineering :) (just joking)

Reply:
> I don't think we should have the word "latest", maybe "industry standard"?
> 
> If anything, we're doing better at reflecting reality by having horrible out-dated systems which are too expensive to replace, as that's what really happens in software engineering :) (just joking)

One could argue that COBOL is an industry standard. I strongly disagree.